docs/tasks/administer-cluster/pvc-protection.md

```diff
@@ -133,6 +131,90 @@ Events:
 ```
 -  Wait until the pod status is `Terminated` (either delete the pod or wait until it finishes). Afterwards, check that the PVC is removed.
 
+### Scenario 3: A pod starts using a PVC that is in Terminating state
```

Comment:
Wasn't this in the other p

Reply:
Yes, I based this PR on the PR https://github.com/kubernetes/website/pull/7164 because it was a prerequisite from my point of view. I'll rebase. I'm sorry for the confusion.